src/Symfony/Component/Mime/Tests/Header/HeadersTest.php

```diff
@@ -247,14 +247,14 @@ public function testToArray()
     public function testInReplyToAcceptsNonIdentifierValues()
     {
         $headers = new Headers();
-        $headers->addHeader('In-Reply-To', 'foobar');
+        $headers->addTextHeader('In-Reply-To', 'foobar');
         $this->assertEquals('foobar', $headers->get('In-Reply-To')->getBody());
     }
 
     public function testReferencesAcceptsNonIdentifierValues()
     {
         $headers = new Headers();
-        $headers->addHeader('References' , 'foobar');
+        $headers->addTextHeader('References', 'foobar');
```

Comment:
why changing this? (apart from the space :) )

Reply:
because tests are failing on this.
There is no addHeader() method at all.

Reply:
Oh ok, please rebase, this has been fixed now

Reply:
Done :)